pkg/template/v2/pkcs12.go

```diff
@@ -107,3 +109,29 @@ func pkcs12certPass(pass, input string) (string, error) {
 func pkcs12cert(input string) (string, error) {
 	return pkcs12certPass("", input)
 }
+
+func pemToPkcs12(cert, key string) (string, error) {
+	return pemToPkcs12Pass(cert, key, "")
+}
+
+func pemToPkcs12Pass(cert, key, pass string) (string, error) {
+	certPem, _ := pem.Decode([]byte(cert))
```

Comment:
I would definitely not ignore the error here.

Reply:
The ignored return value is not an error in this case, but leftover PEM bytes.